README.md

```diff
@@ -1,5 +1,13 @@
 # canvasCourseDeleteUtility
```

Comment:
There should be a description of the overall project here - this could be pulled from the JIRA